config/routes.rb

```diff
@@ -1,117 +1,110 @@
-#                                   accept_transition_screen_api_v2_user PATCH  /api/v2/users/:id/accept_transition_screen(.:format)                                                                                     api/v2/users#accept_transition_screen
-#                                     accept_rollover_screen_api_v2_user PATCH  /api/v2/users/:id/accept_rollover_screen(.:format)                                                                                       api/v2/users#accept_rollover_screen
-#                                               accept_terms_api_v2_user PATCH  /api/v2/users/:id/accept_terms(.:format)                                                                                                 api/v2/users#accept_terms
-#                                                            api_v2_user GET    /api/v2/users/:id(.:format)                                                                                                              api/v2/users#show
-#                                                                        PATCH  /api/v2/users/:id(.:format)                                                                                                              api/v2/users#update
-#                                                                        PUT    /api/v2/users/:id(.:format)                                                                                                              api/v2/users#update
-#                                               api_v2_providers_suggest GET    /api/v2/providers/suggest(.:format)                                                                                                      api/v2/providers#suggest
-#                                                                 api_v2 GET    /api/v2/recruitment_cycles/:recruitment_cycle_year/providers/suggest(.:format)                                                           api/v2/providers#suggest
-#                                                   api_v2_organisations GET    /api/v2/organisations(.:format)                                                                                                          api/v2/organisations#index
-#                                     api_v2_provider_recruitment_cycles GET    /api/v2/providers/:provider_code/recruitment_cycles(.:format)                                                                            api/v2/recruitment_cycles#index
-#                   sync_courses_with_search_and_compare_api_v2_provider POST   /api/v2/providers/:code/sync_courses_with_search_and_compare(.:format)                                                                   api/v2/providers#sync_courses_with_search_and_compare
-#                    sync_with_search_and_compare_api_v2_provider_course POST   /api/v2/providers/:provider_code/courses/:code/sync_with_search_and_compare(.:format)                                                    api/v2/courses#sync_with_search_and_compare
```

Comment:
quick sanity check does frontend call to this has been removed/or need removing?

Reply:
This should be enough - https://github.com/DFE-Digital/publish-teacher-training/pull/958